packages/tldraw/src/Tldraw.tsx

```diff
@@ -504,117 +507,141 @@ const InnerTldraw = React.memo(function InnerTldraw({
-                onDoubleClickShape={app.onDoubleClickShape}
-                onRightPointShape={app.onRightPointShape}
-                onDragShape={app.onDragShape}
-                onHoverShape={app.onHoverShape}
-                onUnhoverShape={app.onUnhoverShape}
-                onReleaseShape={app.onReleaseShape}
-                onPointBounds={app.onPointBounds}
-                onDoubleClickBounds={app.onDoubleClickBounds}
-                onRightPointBounds={app.onRightPointBounds}
-                onDragBounds={app.onDragBounds}
-                onHoverBounds={app.onHoverBounds}
-                onUnhoverBounds={app.onUnhoverBounds}
-                onReleaseBounds={app.onReleaseBounds}
-                onPointBoundsHandle={app.onPointBoundsHandle}
-                onDoubleClickBoundsHandle={app.onDoubleClickBoundsHandle}
-                onRightPointBoundsHandle={app.onRightPointBoundsHandle}
-                onDragBoundsHandle={app.onDragBoundsHandle}
-                onHoverBoundsHandle={app.onHoverBoundsHandle}
-                onUnhoverBoundsHandle={app.onUnhoverBoundsHandle}
-                onReleaseBoundsHandle={app.onReleaseBoundsHandle}
-                onPointHandle={app.onPointHandle}
-                onDoubleClickHandle={app.onDoubleClickHandle}
-                onRightPointHandle={app.onRightPointHandle}
-                onDragHandle={app.onDragHandle}
-                onHoverHandle={app.onHoverHandle}
-                onUnhoverHandle={app.onUnhoverHandle}
-                onReleaseHandle={app.onReleaseHandle}
-                onError={app.onError}
-                onRenderCountChange={app.onRenderCountChange}
-                onShapeChange={app.onShapeChange}
-                onShapeBlur={app.onShapeBlur}
-                onShapeClone={app.onShapeClone}
-                onBoundsChange={app.updateBounds}
-                onKeyDown={app.onKeyDown}
-                onKeyUp={app.onKeyUp}
-                onDragOver={app.onDragOver}
-                onDrop={app.onDrop}
-              />
-            </ErrorBoundary>
-          </ContextMenu>
-          {showUI && (
-            <StyledUI ref={setDialogContainer}>
-              {settings.isFocusMode ? (
-                <FocusButton onSelect={app.toggleFocusMode} />
-              ) : (
-                <>
-                  <TopPanel
-                    readOnly={readOnly}
-                    showPages={showPages}
-                    showMenu={showMenu}
-                    showMultiplayerMenu={showMultiplayerMenu}
-                    showShareMenu={showShareMenu}
-                    showStyles={showStyles}
-                    showZoom={showZoom}
-                  />
-                  <StyledSpacer />
-                  {showTools && !readOnly && <ToolsPanel />}
-                </>
-              )}
-            </StyledUI>
-          )}
-        </StyledLayout>
-      </IntlProvider>
-    </ContainerContext.Provider>
+    <Wrapper
```

Comment:
Could you get rid of the Wrapper component and move these props to StyledLayout?